lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -418,10 +418,45 @@ protected function _getCreateTableSQL($table, array $columns, array $options = a
      */
     public function getListTableIndexesSQL($table, $currentDatabase = null)
     {
-        $table = $this->normalizeIdentifier($table);
-        $table = $this->quoteStringLiteral($table->getName());
+        if (null === $table && !$this->isDatabaseSelected($currentDatabase)) {
```

Comment:
`Connection` is not known within `AbstractPlatform` and children classes, so we cannot use `getParams` here to get the schema name. Getting from USERENV would mean a separate SQL call to the DB which IMHO we should avoid if possible. Accessing `USER*`  views is there to address that.